docs/articles/configs/configoptions.md

```diff
@@ -1,4 +1,5 @@
 ---
+#cspell:ignore configoptions
```

Comment:
ℹ️ In some files where the only issue surfaced was in frontmatter, I used the `#cspell:ignore` technique to ignore it.